tox.ini

```diff
@@ -89,6 +89,7 @@ changedir = docs
 description = check the links in the HTML docs
 extras = docs
 commands =
+    conda update conda
```

Comment:
This will cause more troubles along the line, at least historically this caused a lot of unexpected and unexplained failures.

A better workaround is to use pip only 🤷

Reply:
👍  I'm moving the Windows job to Actions.  I was pleasantly surprised how nice Actions is.